packages/@sanity/cli/src/actions/typegen/generateAction.ts

```diff
@@ -40,8 +41,23 @@ export default async function typegenGenerateAction(
   const trace = telemetry.trace(TypesGeneratedTrace)
   trace.start()
 
-  const codegenConfig = await readConfig(flags.configPath || 'sanity-typegen.json')
+  const codegenConfig = await readConfig(
+    flags.configPath || flags['config-path'] || 'sanity-typegen.json',
```

Comment:
Do we need to be backward compatible? We mark it as experimental/beta so I would say it should be fine to correct this.

```suggestion
    flags['config-path'] || 'sanity-typegen.json',
```

Reply:
Makes sense to me 👍 Will remove the camelcase flag and also add it to the help output